checkbox-ng/checkbox_ng/urwid_ui.py

```diff
@@ -796,6 +806,8 @@ def run(self):
         try:
             return next(i.tp_id for i in self.radio_button_group if i.state)
         except StopIteration:
+            if self._resume_instead:
+                raise ResumeInstead()
```

Comment:
Probably need to check if there is any session to resume before trying to raise this, otherwise it crashes Checkbox (see my main comment).

Reply:
I changed the code so the exception cannot happen if there's no resumable session available. See 24872680d825681dddd6e8c7a85c85f9c682b8fd